packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -92,10 +93,25 @@ export class MarketOperationUtils {
                 this._liquidityProviderRegistry,
             ),
         );
+        const rfqtPromise =
```

Comment:
When would `_opts` ever be undefined?

Can make either make this ternary shorter, or break it out into something else?  This feels a bit jarring to read

One option (still a pretty long statement for ternary tho)
```ts
const rfqtIndicativeQuotes = _opts.rfqt && _opts.rfqt.quoteRequestor ? _opts.rfqt.quoteRequestor.requestRfqtIndicativeQuotesAsync(...) : Promise.resolve<RfqtIndicativeQuoteResponse[]>([]);
```

Another option
```ts
let rfqtIndicativeQuotesPromise = Promise.resolve<RfqtIndicativeQuoteResponse[]>([]);
if (_opts.rfqt && _opts.rfqt.quoteRequestor) {
   rfqtIndicativeQuotesPromise = _opts.rfqt.quoteRequestor.requestRfqtIndicativeQuotesAsync(...)
}
```

Third option (I like this best)
```ts
const rfqtIndicativeQuotes = async (): Promise<RfqtIndicativeQuoteResponse[]> => {
   if (_opts.rfqt && _opts.rfqt.quoteRequestor) {
       return _opts.rfqt.quoteRequestor.requestRfqtIndicativeQuotesAsync(...);
   }
  return [];
}
```

Reply:
I cleaned this up here: https://github.com/0xProject/0x-monorepo/pull/2555/commits/5effc6ec90e5a90402e0efb01cefd0b6a6a8ef1b

Please let me know what you think.